Makefile

```diff
@@ -165,7 +165,11 @@ coverage_html_dir := coverage_html
 # Note: The package version is defined in pywbem/_version.py.
 # Note: Errors in getting the version (e.g. if wheel package is not installed) are
 # detected in _check_version. We avoid confusion by suppressing such errors here.
-package_version := $(shell $(PYTHON_CMD) setup.py --version 2>$(DEV_NULL))
+package_version := $(shell $(PYTHON_CMD) -m setuptools_scm 2>$(DEV_NULL))
```

Comment:
shouldn't line 165 "Note: The package version ... " be removed.

Reply:
I replaced the sentence with: "Note: The package version is automatically calculated by setuptools_scm based on the most recent tag in the commit history, increasing the least significant version indicator by 1."